phpBB/adm/style/admin.js

```diff
@@ -261,5 +261,26 @@ function parse_document(container)
 		$('.actions a:has(i.acp-icon)').mouseover(function () {
 			$(this).css("text-decoration", "none");
 		});
+
+		// Live update BBCode font icon preview
+		const updateIconClass = (element, newClass) => {
+			element.classList.forEach(className => {
+				if (className.startsWith('fa-') && className !== 'fa-fw') {
+					element.classList.remove(className);
+				}
+			});
+
+			element.classList.add(`fa-${newClass}`);
+		};
+
+		const pageIconFont = document.getElementById('bbcode_font_icon');
+
+		pageIconFont.addEventListener('keyup', function() {
```

Comment:
Should check if pageIconFont was found before accessing it, otherwise you produce JS errors on every page